eox_core/edxapp_wrapper/backends/pre_enrollment_l_v1.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Backend for the pre-enrollment (white listings) functionality that works under the open-release/hawthorn.beta1 tag
+"""
+# pylint: disable=import-error, protected-access
+from __future__ import absolute_import, unicode_literals
```

Comment:
I think this is not required in python 3

Reply:
From what I read you're right! I'll remove it